### PR TITLE
player_control: Add format

### PR DESCRIPTION
**Pull request by lasers**
This adds format: string to print (default "{icon}"). 

Ongoing efforts to `FORMAT ALL THE THINGS!` (meme). https://github.com/ultrabug/py3status/issues/98

### CODE REVIEW

File: py3status/modules/player_control.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 Configuration parameters:
     cache_timeout: how often to update in seconds (default 10)
     debug: enable verbose logging (bool) (default False)
+    format: string to print (default "{icon}")
```
Comment (tobes) [inline py3status/modules/player_control.py]: `string to print` feels wrong, we should have a standard text `format of output`?

Comment (lasers) [inline py3status/modules/player_control.py]: D'oh. Copy and paste. Will fix it. Clarify please. Leave it? Rename it to `format of output`? I can't come up with something. `a string to display`, `display string`?

Comment (tobes) [inline py3status/modules/player_control.py]: format of the output

**Comment by ultrabug** [comment]
I like that FORMAT ALL ZE THINGS frenzy, thanks @lasers !

**Pull request merged**